webpack.config.test.js

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const combineLoaders = require("webpack-combine-loaders");
+const nodeExternals = require("webpack-node-externals");
+const path = require("path");
+
+const config = {
+  target: "node",
+  externals: [nodeExternals()],
+  devtool: "cheap-module-source-map",
```

Comment:
+1

Reply:
I keep forgetting webpack expanded the number of devtool options ...